ui/src/pages/backoffice/Acquisition/Order/OrderDetails/OrderDetails.js

```diff
@@ -0,0 +1,220 @@
+                See {order.vendor.name}
+              </Link>
+            </List.Content>
+          </List.Item>
+        </List>
+      </DetailsMenu>
+    );
+  }
+
+  renderDetails = () => {
+    const { data } = this.props;
+    const panels = [
+      {
+        key: 'order-info',
+        title: 'Order information',
+        content: (
+          <Accordion.Content>
+            <div ref={this.orderInfoRef}>
+              <OrderInformation order={data.metadata} />
+            </div>
+          </Accordion.Content>
+        ),
+      },
+      {
+        key: 'payment-info',
+        title: 'Payment information',
+        content: (
+          <Accordion.Content>
+            <div ref={this.paymentInfoRef}>
+              <PaymentInformation order={data.metadata} />
+            </div>
+          </Accordion.Content>
+        ),
+      },
+      {
+        key: 'order-lines',
+        title: 'Order lines',
+        content: (
+          <Accordion.Content>
+            <div ref={this.orderLinesRef}>
+              <OrderLines lines={data.metadata.order_lines} />
+            </div>
+          </Accordion.Content>
+        ),
+      },
+    ];
+    const defaultIndexes =
+      data.metadata.status === 'CANCELLED' ? [0] : [0, 1, 2];
+    const vendor = data.metadata.vendor;
+    const vendorLink = (
+      <Link to={AcquisitionRoutes.vendorDetailsFor(vendor.pid)}>
+        {vendor.name}
+      </Link>
+    );
+    const pid = data.metadata.pid;
+    const details = (
+      <>
+        <label>Order</label> #{pid} <CopyButton text={pid} />
+        <br />
+        <label>Ordered by</label> Someone
+        <br />
+        <label>On</label> {toShortDate(data.metadata.order_date)}
+      </>
+    );
+    return (
+      <Ref innerRef={this.contextRef}>
+        <div className="bo-details">
+          <DetailsHeader
+            title={
+              <>
+                Purchase Order #{data.metadata.pid}{' '}
+                {this.renderStatus(data.metadata.status)}
+              </>
+            }
+            subTitle={<>From {vendorLink}</>}
+            details={details}
+            pid={data.metadata.pid}
+            icon="shopping cart"
+            recordType="Order"
+          />
+          <Container>
+            <Divider />
+            <OrderStatistics order={data.metadata} />
+            <div className="bo-details-content">
+              {this.renderStickyMenu()}
+              <Accordion
+                fluid
+                styled
+                panels={panels}
+                exclusive={false}
+                defaultActiveIndex={defaultIndexes}
+              />
+            </div>
+          </Container>
+        </div>
+      </Ref>
+    );
+  };
+
+  render() {
+    const { isLoading, error, hasError } = this.props;
+    return (
+      <Loader isLoading={isLoading}>
+        <Error error={error}>
+          {!isLoading && !hasError && this.renderDetails()}
```

Comment:
I had issues with the loader here since I'm passing data to the components. If `data` is `null` it's evaluated (but not displayed) so it still results in an error.